docs/USER_RECORD.md

```diff
@@ -234,6 +234,9 @@ optional, when unset the user should not be considered part of any realm. A
 user record with a realm set is never compatible (for the purpose of updates,
 see above) with a user record without one set, even if the `userName` field matches.
 
+`bulkDirectory` → The path to a world-readable copy of the user's bulk directory.
+See [Bulk Directories](HOMED_BULK_DIRS.md) for more details.
```

Comment:
this should declare whether this is an absolute path. Or probably declare that if this is a relative path it should be taken relative to `homeDirectory`

Reply:
as mentioned elsewhere I think we should call this `publicBulkDataDirectory` or so, i.e. clarify the type of data and it's public nature in it.

Reply:
And I think we should clarify that this directory should only contain "simple files" in a shallow list. No subdirs, no symlinks, no weird inodes and stuff.

Reply:
I don't like opening the possibility of it being a relative path... I always pictured this as an absolute path. It will force the client on the other end to do more work for little gain.